Makefile.PL

```diff
@@ -23,7 +23,7 @@ build_requires( 'Test::More' => 0, );
 
 requires(
     'JSON::XS'           => 0,
-    'Locale::TextDomain' => 0,
+    'Locale::TextDomain' => 1.23,
```

Comment:
Shouldn't it be 1.21?

Reply:
1.23 is the lowest version that we have verified. As our instructions are written, everybody will get 1.28 (or higher).